### PR TITLE
✨ Replace loop icon png with custom SF Symbol

### PR DESCRIPTION
**Pull request by caocanx**
Make the icon looks smoother. It created base on the `circle.circle` in SF Symbols 5.

Slide 16_9 - 1

### CODE REVIEW

**Comment by MrKai77** [comment]
Looks good! Just tested this locally and the icon in the settings window looks so much smoother!
My only piece of feedback is that the menubar icon looks a little "heavy". Would you be able to make it look lighter in the menubar? (It looks perfect in the settings window)
Adding `.fontWeight(.light)` should be sufficient :)
Once that is done, this PR should be ready for merge!

File: Loop/MenuBar/MenuBarIconView.swift [hunk 1]
```diff
@@ -10,7 +10,10 @@ import SwiftUI
 struct MenuBarIconView: View {
     @State var rotationAngle: Double = 0.0
     var body: some View {
-        Image(.menubarIcon)
+        Image("loop")
```
Comment (mrkai77) [inline Loop/MenuBar/MenuBarIconView.swift:13]: Add `.fontWeight(.light)`

Comment (caocanx) [reply Loop/MenuBar/MenuBarIconView.swift:13]: Very good advice, I added it.

**Review by MrKai77 — APPROVED** [review]
Looks good!

**Pull request merged**